### ISSUE TITLE
emoji & stickers panel should be hidden while browsing a channel

### ISSUE THREAD

**Issue by VSG24** [telegramdesktop/tdesktop#3271]
It doesn't make any sense to have that big panel visible in the right side while in a read only section such as a channel. It should be hidden.

**Comment by john-preston** [comment]
@VSG24 Yes, perhaps you're right. But right now I've left it always visible so that the chat and background won't jump when you go from a chat to a channel and back.

**Comment by stek29** [comment]
@john-preston maybe it should automatically open channel info there, once it supports showing profiles too.
